.github/workflows/plugins-ci.yml

```diff
@@ -13,6 +13,16 @@ on:
         required: false
         default: false
         type: boolean
+      pnpm:
```

Comment:
I'm not sure the fastify org needs this option.

You can still add a package.json's script `pretest: "npm install pnpm@7.4.0 -g"`

Reply:
I think we just need to support npm / pnpm / yarn. There shouldn't be any need to support anything else.

Because apart from that, it's basically all project dependency-related stuff, but package management tools are obviously not in that category.